### DISCUSSION TITLE
Documentation on issue #1208

### DISCUSSION THREAD

**Discussion by PaulKruger** [Azure/azure-functions-durable-extension#1953] [Q&A] [answered]
Issue #1208 mentions that there is documentation surrounding the IMessageSerializerSettingsFactory in durable functions; however, I am unable to locate it.  If someone is familiar with this issue, can you provide a link to this documentation?

**Comment by cgillum** [comment, marked as answer]
I think this is the documentation you're looking for: https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-serialization-and-persistence?tabs=csharp#customizing-serialization-and-deserialization

/cc @ConnorMcMahon

**Reply by PaulKruger** [reply]
That appears to partially address my issues.  

Recently I began attempting to move error handling within orchestrations rather than activity functions, and sql related error handling is throwing serialization issues.  I was hoping the documentation might address common scenarios such a sqlexception or custom exceptions that inherit from the base exception class.  

For example, in my case I'm using a fan out / fan in method to execute a number of queries against a database, and orchestration needs to be able to handle whether to retry or perform some other activity.

Are there any examples or documentation that more specifically address these concerns?

**Comment by PaulKruger** [comment]
In my case, I eventually made the switch to Microsoft.Data.SqlClient from System.Data.SqlClient and that fixed the issue for me.
